src/main/java/mate/academy/controller/AuthenticationController.java

```diff
@@ -14,17 +20,36 @@
 public class AuthenticationController {
     private final AuthenticationService authenticationService;
     private final UserMapper userMapper;
+    private final JwtTokenProvider jwtTokenProvider;
 
     public AuthenticationController(AuthenticationService authenticationService,
-                                    UserMapper userMapper) {
+                                    UserMapper userMapper,
+                                    JwtTokenProvider jwtTokenProvider) {
         this.authenticationService = authenticationService;
         this.userMapper = userMapper;
+        this.jwtTokenProvider = jwtTokenProvider;
     }
 
     @PostMapping("/register")
-    public UserResponseDto register(@RequestBody @Valid UserRegistrationDto userRequestDto) {
+    public ResponseEntity<Object> register(@RequestBody @Valid UserRegistrationDto userRequestDto) {
```

Comment:
you shouldn't return token in the response to registration, only in /login, return the original version